config/develop/example-dev-project.j2

```diff
@@ -1,28 +1,40 @@
-stack_name: example-dev-project
+{% set stack_name = "example-dev-project" %}
+{% set read_write_users = ["bruno.grande@sagebase.org"] %}
+{% set read_only_users = ["tess.thyer@sagebase.org", "thomas.yu@sagebase.org"] %}
 
-stack_tags:
-  Department: IBC
-  Project: Infrastructure
-  OwnerEmail: nextflow-admins@sagebase.org
+# DO NOT EDIT ANYTHING BELOW THIS LINE -----------------------------------------------------------
+
+template_path: tower-project.yaml
+stack_name: {{ stack_name }}
+dependencies:
+  - common/nextflow-forge-iam-policy.yaml
```

Comment:
It was added originally because of some `!ImportValue` calls in the template. These are gone now, so I got rid of the dependency. That said, the VPC obviously needs to exist before this project. 

I opted for adding the `develop/nextflow-ecs-service.yaml` dependency, which is technically more correct and implies the dependency on the VPC.